### PR TITLE
Update Toolkit

### PR DESCRIPTION
**Pull request by MGatner**
Updates to the latest development toolkit. For some reason reverting to the default test database prefix ("`db_`") broke the tests so I re-applied the `""` version - probably some deeper issue causing that which should be looked at sometime.

Note that this includes the new CodeIgniter Coding Standard but does not apply it since that would change almost every file - I will do that in a subsequent PR.

### CODE REVIEW

**Comment by MGatner** [comment]
@lonnieezell I don't have access to this repo. In order to get Coveralls working you will need to add a repo secret "COMPOSER_AUTH" with your GitHub token, e.g.:
```
{
    "github-oauth": {
        "github.com": "ghp_MySuperSecretGitHubAccessToken"
    }
}
```

... then go to Coveralls.io and import this repo. In the meantime the workflow failure can be ignore since all the actual tests passed.

File: .php-cs-fixer.dist.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+$options = [
+    'finder'    => $finder,
+    'cacheFile' => 'build/.php-cs-fixer.cache',
+];
+
+/* Reenable after incremental changes are applied
+return Factory::create(new Standard(), $overrides, $options)->forLibrary(
+    'Library',
+    'Tatter Software',
```
Comment (lonnieezell) [inline .php-cs-fixer.dist.php:24]: Is this portion still relevant?

Comment (MGatner) [reply .php-cs-fixer.dist.php:24]: Gah, no. 🤦‍♂️ Good catch. I basically apply my templates and then go through and remove all the LLC stuff. I will rebase this out.

**Comment by MGatner** [comment]
Also please be sure to check out **SECURITY.md** - my default (modeled after CrowPHP) adapted for here, but I would like to start applying this to all our CodeIgniter repos.

**Review by lonnieezell — APPROVED** [review]

**Comment by lonnieezell** [comment]
> Also please be sure to check out SECURITY.md - my default (modeled after CrowPHP) adapted for here, but I would like to start applying this to all our CodeIgniter repos.

I saw that and it looks fine. We have something similar on CI 4 I think. I do need to make sure that email address is setup and working, of course. :)

**Pull request merged**